src/stripe.ios.ts

```diff
@@ -35,6 +35,38 @@ export class Stripe {
     );
   }
 
+  createSource(card: CardCommon, cb: (error: Error, source: Source) => void): void {
+    if (!card) {
+      if (typeof cb === 'function') {
+        cb(new Error('Invalid card'), null);
+      }
+      return;
+    }
+    
+    const sourceParams = STPSourceParams.cardParamsWithCard(card.native)
+
+    const apiClient = STPAPIClient.sharedClient();
+    apiClient.createSourceWithParamsCompletion(
+      sourceParams, callback(cb, (source) => <Source> {
+        id: source.stripeID,
+        amount: source.amount,
+        card: card,
+        client_secret: source.client_secret,
+        created: source.created,
+        currency: source.currency,
+        flow: source.flow,
```

Comment:
Enums on iOS are a real pain. They come through as integers, which need to be mapped to the proper string values.